OJP_Trips.xsd

```diff
@@ -548,6 +547,11 @@
 					</xs:sequence>
 				</xs:complexType>
 			</xs:element>
+			<xs:element name="LineIdAtBoarding" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Identifier of the boarded line/service at the boarding. This is not a reference. This identifier is used to recognize in a distributed environment, that two systems refer to the same line service (or service) while using their own internal references.</xs:documentation>
```

Comment:
> Identifier of the boarded line/service _during_ boarding. This is not a reference. This identifier is used to recognize _the line_ in a distributed environment _(two_ systems _referring_ to the same line service (or service) while using their own internal references).

With the current (slightly changed) documentation text it is still not clear to me what to expect in/from this element. Maybe enhance it a bit more?

What is meant by "line service (or service)"?